app/src/main/java/org/simple/clinic/drugs/selection/entry/confirmremovedialog/ConfirmRemovePrescriptionDialog.kt

```diff
@@ -57,7 +64,7 @@ class ConfirmRemovePrescriptionDialog : AppCompatDialogFragment(), UiActions {
   }
 
   private fun setupDialog(): Observable<UiChange> {
-    return Observable.merge(screenDestroys, removeClicks())
```

Comment:
We cannot remove `screenDestroys` here because it is used in the controller (see [source](https://github.com/simpledotorg/simple-android/blob/032ccc500c98295b1a67ad641c9fd976be4f8692/app/src/main/java/org/simple/clinic/ReplayUntilScreenIsDestroyed.kt) of `ReplayUntilScreenIsDestroyed`) to dispose the stream automatically.

Since you are on leave, I will make this correction.